bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ZeebeCalledElementValidator.java

```diff
@@ -30,13 +37,21 @@
   @Override
   public void validate(
       final ZeebeCalledElement element, final ValidationResultCollector validationResultCollector) {
-    final String processId = element.getProcessId();
 
-    if (processId == null || element.getProcessId().isEmpty()) {
-      validationResultCollector.addError(
-          0,
-          String.format(
-              "Attribute '%s' must be present and not empty", ZeebeConstants.ATTRIBUTE_PROCESS_ID));
+    final Optional<String> processId =
+        Optional.ofNullable(element.getProcessId()).filter(notEmpty());
+    final Optional<String> processIdExpression =
+        Optional.ofNullable(element.getProcessIdExpression()).filter(notEmpty());
+
+    if (processId.isPresent() && processIdExpression.isPresent()) {
```

Comment:
The two cases of this if are equivalent to:

```suggestion
    if (processId.isPresent() == processIdExpression.isPresent()) {
```